htdocs/web_portal/views/downtime/confirm_add_downtime.php

```diff
@@ -38,30 +53,38 @@
         //echo date_format($timestamp, 'l jS \of F Y \a\t\: h:i A');
         xecho($params['DOWNTIME']['END_TIMESTAMP']);
     ?></li>
-    <?php
-    if(count($services > 1)){
-        echo "<li><b>Affecting Services:</b>";
-    }else{
-        echo "<li><b>Affecting Service:</b>";
-    }
-    ?>
-        <ul>
+
+    <?php foreach ($serviceWithEndpoints as $siteID => $siteDetails) : ?>
         <?php
-        foreach($services as $id){
-            $service = \Factory::getServiceService()->getService($id);
-            $safeHostName = xssafe($service->getHostname());
-            echo "<li>" . $safeHostName . "</li>";
-            }
+         $siteName = $params['SITE_LEVEL_DETAILS'][$siteID]['siteName'];
+
+         echo '<li><strong>Site Name: </strong>';
+         echo $siteName;
+         echo '</li>';
         ?>
-        </ul>
-    </li>
-    <?php
-    if(count($endpoints > 1)){
-        echo "<li><b>Affecting Endpoints:</b>";
-    }else{
-        echo "<li><b>Affecting Endpoint:</b>";
-    }
-    ?>
+
+        <ul>
+            <li>
+                <b>Affecting Service and Endpoint(s):</b>
+
+                <?php foreach ($siteDetails as $serviceID => $data) : ?>
+                    <?php
+                    $endpoints = $data['endpoints'];
+                    $service = \Factory::getServiceService()
+                                    ->getService($serviceID);
+            $safeHostName = xssafe($service->getHostname());
```

Comment:
This line is under-indented.

Reply:
<img width="908" alt="Intendation" src="https://github.com/GOCDB/gocdb/assets/127398882/3e6b4fe9-fe25-48a9-89f6-cddd650107b2">

Again, VSCODE Doesn't complained about it.  Not sure why